Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1196,14 +1196,14 @@ public function isBinaryOperator($index)
                 T_XOR_EQUAL => true,            // ^=
             );
             if (defined('T_POW')) {
-                $arrayOperators[T_POW] = true;          // **
-                $arrayOperators[T_POW_EQUAL] = true;    // **=
+                $arrayOperators[T_POW] = true; // **
+                $arrayOperators[T_POW_EQUAL] = true; // **=
```

Comment:
Not sure, because the fixer works more generic now with the semicolons. For example it will do this which I don't think is in PSR2

```php
<?php
    test();     $a = 4;

```

```php
<?php
    test(); $a = 4;
                
```